### PR TITLE
Add gRPC to oss-fuzz projects

### PR DESCRIPTION
**Pull request by matt-kwong**
Not ready to merge; see #374

### CODE REVIEW

**Comment by kcc** [comment]
I'm glad to see this, thanks! 
One important thing missing is the seed corpora (https://github.com/google/oss-fuzz/blob/master/docs/new_project_guide.md#seed-corpus) but it may come in the following changes. 

I also see #374. Let's merge and investigate further.

File: projects/grpc/build.sh [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+bazel-bin/test/core/end2end/*.a \
+bazel-bin/external/submodule_boringssl/*.so \
+bazel-bin/external/submodule_zlib/_objs/z/external/submodule_zlib/*.o \
+bazel-bin/third_party/nanopb/*.so \
+bazel-bin/*.a \
+"
+
+# build project
+bazel
+BAZEL build --spawn_strategy=standalone --genrule_strategy=standalone :all test/... examples/cpp/...
```
Comment (kcc) [inline projects/grpc/build.sh]: is this BAZEL or bazel?

Comment (matt-kwong) [inline projects/grpc/build.sh]: Woops, this is supposed to be bazel. I did have as bazel when I tested locally but forgot to commit the change in my fork. I'll open another PR to fix this.

**Comment by kcc** [comment]
next problem: I see that your fuzzer binaries are dynamically linked to bazel-bin/external/submodule_boringssl/libcrypto.so
oss-fuzz does not support dynamically-linking to anything but the basic system libs.
Is it possible for you to link these statically?

**Comment by kcc** [comment]
See https://github.com/google/oss-fuzz/blob/master/docs/fuzzer_environment.md for some explanations

**Comment by kcc** [comment]
Next problem, which seems to only affect `ssl_server_fuzzer`: 
```
==6==ERROR: AddressSanitizer: ABRT on unknown address 0x000000000006 (pc 0x7fccb667c428 bp 0x7fff356080d0 sp 0x7fff35607ce8 T0)
SCARINESS: 10 (signal)
    #0 0x7fccb667c427 in gsignal (/lib/x86_64-linux-gnu/libc.so.6+0x35427)
    #1 0x7fccb667e029 in abort (/lib/x86_64-linux-gnu/libc.so.6+0x37029)
    #2 0x513fa4 in LLVMFuzzerTestOneInput /src/grpc/test/core/security/ssl_server_fuzzer.c:91:3
```

The problem is that the code does 
```
  GPR_ASSERT(GRPC_LOG_IF_ERROR("load_file",
                               grpc_load_file(SSL_CA_PATH, 1, &ca_slice)));

```
but these files are not shipped to ClusterFuzz. 

You may reproduce the failure like this:
```
python infra/helper.py --nopull run_fuzzer   grpc  ssl_server_fuzzer
```

Is it possible to embed the SSL data into the code, like most of the openssl/boringssl/etc fuzz targets to?

**Pull request merged**